go/vt/vttablet/tabletserver/schema/load_table.go

```diff
@@ -157,3 +171,39 @@ func getNum(in map[string]string, key string) (int, error) {
 	}
 	return v, nil
 }
+
+func getCols(in map[string]string, key string) ([]string, error) {
```

Comment:
I feel like this should be called `parseMessageFields`?  Minor thing, I just didn't realize what it was for at first. We could also address that with a function comment.

Reply:
done